SUMMARY.md

```diff
@@ -12,12 +12,12 @@
   * [Add a contributor](getting-started/first-steps.md#add-a-contributor)
 * [Quick tips](getting-started/quick-tips.md)
 * [Webinars](getting-started/webinars.md)
-* [FAQ](getting-started/faq.md)
```

Comment:
@AlexVipond why'd you drop the FAQ here?

Reply:
Deleting this one as the questions inside it didn't feel like they were frequently asked, and the ones that were compelling are already answered in more searchable articles in the FAQ section